raiden_libs/test/fixtures/contracts.py

```diff
@@ -0,0 +1,126 @@
+import pytest
+import logging
+from web3.utils.events import get_event_data
+from eth_utils import is_address
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def contracts_manager():
+    """Reimplement this - fixture should return raiden-lib's contract manager.
+        (return ContractManager(CONTRACTS_SOURCE_DIRS))
+    """
+    raise NotImplementedError(
+        "In order to use contract fixtures please override contracts_manager fixture."""
```

Comment:
Use single quotes here. Also I guess you did not want to close it with a triple quote ;)